docs/pages/2019_MARVEL_Psik_MaX/sections/verdi_cmdline.rst

```diff
@@ -39,406 +54,385 @@ The first output line should look like
+For a more detailed list of properties, you can use ``verdi process show``, but to address any specific process, you need an identifier for it.
+An identifier for any entity in AiiDA comes in three different forms:
+
+ * PK
+ * UUID
+ * LABEL
+
+The first one, the PK or primary key, you have already seen.
+These are the integers in the first column of the output printed by ``verdi process list``.
+The PK is a unique identifier for any entity, but only for that *entity type* and *within your database*.
+That is why, in addition to their PK, all AiiDA entities also have a UUID (Universal Unique Identifier), which as the name suggests, should be unique, `even across databases`, and will never change.
+That is to say, if you share some of your data with somebody else, the UUIDs will remain identical, unlike the PKs which will most likely change.
+
+.. note::
+
+    Any ``verdi`` parameter that expects an identifier will accept either a PK, a UUID or a label, although, not all entities will have a label, as it is an optional field.
+    For a UUID you can even specify only a part of it, as long as it starts at the beginning and the partial can be uniquely resolved.
+    For more information on identifiers in ``verdi`` and AiiDA in general, refer to the `documentation online <https://aiida-core.readthedocs.io/en/latest/verdi/verdi_user_guide.html#cli-identifiers>`_.
+
+Let's now consider the process with the UUID ``ce81c420-7751-48f6-af8e-eb7c6a30cec3``.
+Use this identifier to get more information about it:
 
 .. code:: console
 
-    verdi process show <pk>
+    verdi process show <IDENTIFIER>
 
-The output depends on the specific pk chosen and should inform you about
-the input nodes (e.g. pseudopotentials, kpoints, initial structure,
-etc.), and output nodes (e.g. output structure, output parameters,
-etc.).
+.. note::
 
-**PKs/IDs vs. UUIDs**: Beside the (integer) PK, very convenient to
-reference a calculation or data node in your database, every node has a
-UUID (Universally Unique ID) to identify it, that is preserved even when
-you share some nodes with coworkers—while the PK will most likely
-change. You can see the UUID in the output of ``verdi process show``
-or ``verdi node show``. Moreover, if you have already a UUID and you
-want to get the corresponding PK in your database, you can use
-``verdi node show <UUID>``, as we are going to do now.
+    For the remainder of this section, when commands are displayed, any fields enclosed in angular brackets, such as ``<IDENTIFIER>``, are placeholders to be replaced by the actual value of that field.
 
-Let us now consider the node with
-``UUID = ce81c420-7751-48f6-af8e-eb7c6a30cec3``, which identifies a
-relaxation of a BaTiO\ :sub:`3` unit cell run with Quantum Espresso ``pw.x``.
-You can check the information on this node and get the PK with:
+Again, since the UUID is unchanging we know what it represents: a Quantum Espresso ``pw.x`` relaxation of a BaTiO\ :sub:`3` unit cell.
```

Comment:
```suggestion
Again, since the UUID is universally unique, we know what it represents in your database: a Quantum Espresso ``pw.x`` relaxation of a BaTiO\ :sub:`3` unit cell.
```

Reply:
done